core/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 0.42.0 -- unreleased
```

Comment:
cc @jxs Seems to be present in other PRs already? See https://github.com/libp2p/rust-libp2p/blob/master/docs/release.md#make-changelog-entries.

Reply:
I was only adding this because I already saw it in at leat one CHANGELOG.md

Reply:
Just in case @jxs wants to change that, the command is:
```bash
rg " -- unreleased" --files-with-matches | xargs sed -i '' 's/-- unreleased//g'
```
if you are using BSD sed
and 
```bash
rg " -- unreleased" --files-with-matches | xargs sed -i 's/-- unreleased//g'
```
for GNU sed